src/Resources/config/makers.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="false"/>
```

Comment:
is it best practice for `makers.xml` IMO we should mark it public=false on the `<service>`-node

Reply:
this is how symfony has done it, please refer https://github.com/symfony/maker-bundle/blob/master/src/Resources/config/makers.xml